test/helpers/pkcs12.c

```diff
@@ -18,8 +18,8 @@
 #include <openssl/x509v3.h>
 #include <openssl/pem.h>
 
-#include "testutil.h"
-#include "pkcs12_helper.h"
+#include "../testutil.h"
+#include "./pkcs12.h"
```

Comment:
Was `./` necessary?

Reply:
It turned out not to be necessary, but I think it's good to make clear that this `#include` is not for `include/openssl/pkcs12.h`-

Reply:
You gotta admit that it kinda sticks out, when the rest of the source has "filename.h" without any directory part to mean the header file in the same directory as the source, `"e_os.h"` being the unique exception...

Reply:
Yes, this - purposely - points out which `pkcs12.h` is meant.
Shall I still remove the `./`?

Reply:
Frankly, [shrug].  I just gave you my thoughts, and leave it to you do decide what you want to do.  For the compiler, this makes zero difference.  If you think this makes a difference for humans reading the source, then fine

Reply:
Fine, then I'd say let's keep it for clarity.

Reply:
I think a comment is more clear and also less confusing.

Reply:
> I think a comment is more clear and also less confusing.

All right, done.